doc-content/enterprise-only/processes/interacting-with-processes-bulk-actions-on-tasks-con.adoc

```diff
@@ -0,0 +1,6 @@
+[id='interacting-with-processes-bulk-actions-on-tasks-con']
+= Bulk actions on tasks
+
+In the *Tasks* and *Task Inbox* pages in {CENTRAL}, you can perform bulk actions over multiple tasks in a single operation.
+
+NOTE: If a specified bulk action is not permitted based on the task status, a notification is displayed and the operation is not executed on that particular tasks. 
```

Comment:
on that particular tasks. -> on that particular task.

Reply:
@kaldesai +1 to 'that particular task.'